.github/workflows/index-data.yml

```diff
@@ -0,0 +1,36 @@
+name: index-data
+
+on:
+  workflow_call:
+    inputs:
+      projects:
+        required: true
+        type: string
+
+env:
+  TEA_PANTRY_PATH: ${{ github.workspace }}/pantry
+
+jobs:
+  queue_detail_ingestion:
+    runs-on: ubuntu-latest
+    needs: [bottle, upload]
+    steps:
+      - name: co pantry
+        uses: actions/checkout@v3
+        with:
+          path: pantry
+          repository: teaxyz/pantry.core
+
+      - uses: teaxyz/setup@v0
+        with:
+          # necessary because we currently require a `.git` directory
+          srcroot: pantry
+
+      - run: ./scripts/index-packages.ts ${{ inputs.projects }}
+        working-directory: pantry
```

Comment:
If you look at the cleanup, this can be much reduced now:
```yaml
- uses: actions/checkout@v3
- uses: teaxyz/setup@v0
- run: ./scripts/index-packages.ts ${{ inputs.projects }}
```

Reply:
applied.